### PR TITLE
Sd425 helper sphinx docs

### PR DESCRIPTION
**Pull request by jstucke**
New sphinx docs
- install requirements in `docsrc/doc_requirements.txt`
- build with `docsrc/build.sh` or `make -C docsrc html`

### CODE REVIEW

File: docsrc/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```
Comment (dorpvom) [inline docsrc/build.sh:1]: Is this not outdated now? I think we have the github actions for that

File: docsrc/conf.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Configuration file for the Sphinx documentation builder.
```
Comment (dorpvom) [inline docsrc/conf.py:1]: No idea how to review this ..

File: src/helperFunctions/uid.py [hunk 3]
```diff
@@ -1,46 +1,47 @@
...
 
 from helperFunctions.dataConversion import make_bytes
 from helperFunctions.hash import get_sha256
 
 
-def create_uid(input_data):
+def create_uid(input_data: AnyStr) -> str:
     '''
-    creates an unique identifier: SHA256_SIZE
+    generate a UID (unique identifier) SHA256_SIZE for a string containing data
```
Comment (dorpvom) [inline src/helperFunctions/uid.py:10]: Hm. I thought we'd encounter bytes here not strings. What would the sense be to generate this based on a string?

**Review by dorpvom — CHANGES_REQUESTED** [review]
Maybe there's already legacy stuff here? Do we even need the stuff in docs (aside from the pdfs) anymore?

File: src/helperFunctions/uid.py (hunk 3 above)
Comment (jstucke) [reply src/helperFunctions/uid.py:10]: this is actually used e.g. in `db_interface_frontend_editing.py` with a string but changing that is probably out of scope. Changed the doc string

**Review by dorpvom — APPROVED** [review]

**Pull request merged**